src/accessors.jl

```diff
@@ -6,27 +6,20 @@ import ..AstroDates: DateTime, year, month, day,
 
 export timescale
 
-timescale(ep::Epoch{S}) where {S} = S
+timescale(ep::Epoch) = ep.scale
 
 function DateTime(ep::Epoch)
-    if !isfinite(ep.offset)
-        if ep.offset < 0
+    if !isfinite(ep.fraction)
+        if ep.fraction < 0
             return DateTime(AstroDates.MIN_EPOCH, AstroDates.H00)
         else
             return DateTime(AstroDates.MAX_EPOCH, Time(23, 59, 59.999))
         end
     end
 
-    sum = ep.offset + ep.ts_offset
-    o′ = sum - ep.ts_offset
-    d′ = sum - o′
-    Δo = ep.offset - o′
-    Δd = ep.ts_offset - d′
-    residual = Δo + Δd
-
-    carry = floor(Int64, sum)
-    offset2000B = (sum - carry) + residual
-    offset2000A = ep.epoch + carry + Int64(43200)
+    carry = floor(Int64, ep.fraction)
+    offset2000B = ep.fraction - carry
```

Comment:
I think this is similar to
```julia
offset2000B, carry = modf(ep.fraction)
```
but I guess you really want to have `carry` as an integer?

Reply:
I think this whole code path should be unnecessary. `ep.fraction < 1.0` should hold by definition. Will check 👍